README.rst

```diff
@@ -68,6 +98,20 @@ directly, but instead be manually inspected. E.g.:
    Note that there isn't a comma in the end of the line. The last argument is
    treated as the reason why a suggestion cannot be automatically applied.
 
+Updating
+--------
+To stay current with codespell developments it is possible to build codespell from GitHub via::
+
+    ``pip install --upgrade git+https://github.com/lucasdemarchi/codespell.git``
+
+**Important Notes:**
+* Sometimes installing via `pip` will complain about permissions. If this is the case then run with ::
+
+    ``pip install --user --upgrade git+https://github.com/lucasdemarchi/codespell.git``
+
+* It has been reported that after installing from `pip`, codespell can't be located. Please check the $PATH variable to see if `~/.local/bin` is present. If it isn't then add it to your path. 
```

Comment:
and here, `pip` and `~/local/bin`

Reply:
see how they don't render properly

https://github.com/luzpaz/codespell/blob/ace6c0d9b84c74fa9fff698bdab6763b90cd1ae8/README.rst